### ISSUE TITLE
Allow setting up component from web interface

### ISSUE THREAD

**Issue by bezmi** [bezmi/hass_custom_components#7]
Currently the only way to set up the component it to edit the yaml. We need to add the necessary stuff to allow setup from the web interface.

**Comment by roninf** [comment]
If i can be of any help let me know...unfortunately i do not know what needs to be done....

**Comment by bezmi** [comment]
Basic functionality is mostly implemented on the current master branch. The config flow allows setting the same values as `configuration.yaml` in v2.0.0. Before releasing v2.1.0, there are a few things to do:

1. Implement an options flow, so settings may be changed later without having to delete and reconfigure the integration
2. For now, there is a JVC Projector device, with a single remote entity attached to it. I would like to expose the attributes (power_state, input, picture_mode, etc) from the remote entity as sensor entities attached to the JVC Projector device to simplify automation for new users. The remote entity should remain untouched (keep all of the attributes), so the integration doesn't break for people on v2.0.0.

I'm not sure what your experience level is with python/home assistant, but the plan is above :)

**Comment by roninf** [comment]
I think i would need to learn Python first... no clue about it. Dont know if thats a leaning from scratch by doing thing...

**Comment by bezmi** [comment]
Yes. If you use the component, then consider adding any information about your automations to the discussion board to help other users, or update the documentation to be more helpful. These are both great ways of contributing.
